sdk/ai/azure-ai-client/samples/agents/user_functions.py

```diff
@@ -0,0 +1,66 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+import json
+import datetime
+
```

Comment:
Perhaps put some comments here saying these are helper functions used by the Agents sample code.